### PR TITLE
remove babel 5x warnings; remove empty bower;

### PR DESCRIPTION
**Pull request by eriktrom**
Removes babel 5x warning
Removes exit code 1 when running tests locally w/o bower installed globally

cc @rwwagner90

### CODE REVIEW

**Comment by eriktrom** [comment]
ps - w/o updating ember-cli-changelog upstream, optionalDependencies was used to remove the babel 5x warning

(same as https://github.com/html-next/ember-gestures/pull/111)

**Comment by rwwagner90** [comment]
We may want to just remove ember-cli-changelog. I don't have the time to rewrite it.

**Comment by eriktrom** [comment]
> We may want to just remove ember-cli-changelog. I don't have the time to rewrite it.

done, on both repo's

File: package.json [hunk 1]
```diff
@@ -66,5 +65,6 @@
   },
   "ember-addon": {
     "configPath": "tests/dummy/config"
-  }
+  },
+  "optionalDependencies": {}
```
Comment (runspired) [inline package.json]: Did you mean to move ember-cli-changelog here? regardless, we really should update it soon :D

Comment (eriktrom) [reply package.json]: https://github.com/html-next/ember-hammertime/issues/58 <-- at least we'll track it for tonight

**Comment by eriktrom** [comment]
i can't publish this repo, just fyi

**Pull request merged**